### ISSUE TITLE
React 16

### ISSUE THREAD

**Issue by zslabs** [FormidableLabs/component-playground#104]
👋  Wooohoo, I'm the first schmuck to ask about React 16!!

Being the latest and greatest, I decided to give updating both `react` and `react-dom` to `16.0.0`, but am getting a few errors when attempting to compile, such as:

```
ERROR in ./node_modules/component-playground/node_modules/react-dom/lib/ReactCompositeComponent.js
Module not found: Error: Can't resolve 'react/lib/ReactCurrentOwner' in '/Users/zach/Sites/rhinostyle/node_modules/component-playground/node_modules/react-dom/lib'
```

```
ERROR in ./node_modules/component-playground/node_modules/react-dom/lib/traverseAllChildren.js
Module not found: Error: Can't resolve 'react/lib/ReactCurrentOwner' in '/Users/zach/Sites/rhinostyle/node_modules/component-playground/node_modules/react-dom/lib'
```

This is most likely just due to some of the changes FB mentioned in [their release post](https://facebook.github.io/react/blog/2017/09/26/react-v16.0.html) today, but just wanted to make sure we had a note of it here to help determine LOE to update for compatibility. Thanks as always!

**Comment by kenwheeler** [comment]
If you delete component-playgrounds node_modules, it should work. I'll get the deps on this fixed so that it doesn't install that.

**Comment by zslabs** [comment]
Weird, I had previously done that with no luck `¯\_(ツ)_/¯`

**Comment by syyvstar** [comment]
I meet this issue too.  And I think also should  update the webpack version. I  give updating both react and react-dom to 16.0.0 and I delete the  component-playgrounds node_modules, occuring the error. 

![image](https://user-images.githubusercontent.com/30794902/30999421-b82bf7b6-a507-11e7-9e52-4b73a1a7b02e.png)

**Comment by kenwheeler** [comment]
Closing, fixed in 3.0
